charts/stable/haste/Chart.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v2
+appVersion: 2021.04.18
+description: haste helm package
+name: haste
+version: 1.1.0
```

Comment:
Should probably start at 1.0.0

Reply:
updated in https://github.com/k8s-at-home/charts/pull/800/commits/769f44f54a9f96ea280834e666a3c0e2d46314b0